### PR TITLE
Fix staging

### PR DESCRIPTION
**Pull request by fcarelse**
Got staging of development and demo environments working for both api and frontend

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,8 +35,11 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
+    "prestart": "bash -c \"cp public/.htaccess.webdev public/.htaccess\"",
```
Comment (Kivylius) [inline package.json:40]: not sure bash is correct here, may not work on unux?

File: public/.htaccess [hunk 2]
```diff
@@ -0,0 +1,9 @@
+RewriteEngine On
+
+#RewriteRule ^index.html$ http://host.docker.internal:3000/ [NE,NC,P,L,QSA]
+
+#RewriteCond %{REQUEST_FILENAME} !-d
+#RewriteCond %{REQUEST_FILENAME} !-f
+RewriteRule ^([^.]+)$ http://host.docker.internal:3000/#/$1 [NE,NC,P,L,QSA]
+
+RewriteRule ^(.+)$ http://host.docker.internal:3000/$1 [NE,NC,P,L,QSA]
```
Comment (Kivylius) [inline public/.htaccess:9]: beleve this is somethin not be part of "client"

File: public/.htaccess.webdev [hunk 3]
```diff
@@ -0,0 +1,9 @@
+RewriteEngine On
+
+#RewriteRule ^index.html$ http://host.docker.internal:3000/ [NE,NC,P,L,QSA]
+
+#RewriteCond %{REQUEST_FILENAME} !-d
+#RewriteCond %{REQUEST_FILENAME} !-f
+RewriteRule ^([^.]+)$ http://host.docker.internal:3000/#/$1 [NE,NC,P,L,QSA]
```
Comment (Kivylius) [inline public/.htaccess.webdev:7]: and this

File: src/helper/Constants/Constants.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const URL_LOGIN = "/api/user/login";
+export const URL_LOGOUT = "/api/user/logout";
+export const URL_LOGGEDIN = "/api/user/loggedIn";
+
+export const URL_HELLO = "/api/rest/hello";
+export const URL_TODOS = "/api/rest/todos";
+export const URL_TODOLIST = "/api/rest/todolist";
```
Comment (Kivylius) [inline src/helper/Constants/Constants.ts]: this should be an enum, alose do you have a react-router?

File: src/pages/Todolist/TodolistPage.tsx [hunk 5]
```diff
@@ -1,3 +1,36 @@
...
+import { useEffect, useState } from "react";
+import { URL_TODOS } from "../../helper/Constants/Constants";
+import { fetchData } from "../../helper/Fetch/FetchHelper";
 
-export const TodolistPage = ({}) => <div>Todolist Page</div>;
+export const TodolistPage = ({}) => {
+  const [todos, setTodos] = useState<Array<Record<string, any>>>([]);
+  useEffect(() => {
+    (async () => {
+      const newTodos = await fetchData({ url: URL_TODOS });
```
Comment (Kivylius) [inline src/pages/Todolist/TodolistPage.tsx]: i beleve this can wait, should be in try catch.

File: stage/stage.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	Stage ok!
```
Comment (Kivylius) [inline stage/stage.html]: you should not have any `stage` (or any env specific tbh) code commited, the code should be build for stage using some config file or .env file, may need to do a bit more research on evimrents

Comment (fcarelse) [reply stage/stage.html]: stage no longer committed

**Pull request merged**